### ISSUE TITLE
Check architecture tests

### ISSUE THREAD

**Issue by koppor** [koppor/jabref#352]
Why can JavaFX be imported in the logic package without alarms going off?

Refs https://github.com/JabRef/jabref/pull/5470

**Comment by matthiasgeiger** [comment]
Therefore:
https://github.com/JabRef/jabref/blob/bc26c3ac5a3f3e0e7d806ac3639bf079c98441e0/src/test/java/org/jabref/architecture/MainArchitectureTests.java#L49

I don't remember where and when this was discussed but the outcome was that using the FXcollections should be allowed in logic.

**Comment by koppor** [comment]
Yeah, I remember the long discussions in the dev call. I was on the side to allow them 🙈. Should rethink it now that Eclipse Collections (https://github.com/JabRef/jabref/issues/3023)
 is mature.
